trin-execution/src/storage/evm_db.rs

```diff
@@ -195,13 +193,20 @@ impl EvmDB {
     fn commit_storage_changes(
         &mut self,
         address_hash: B256,
-        rocks_account: Option<RocksAccount>,
         storage: Vec<(U256, U256)>,
     ) -> anyhow::Result<()> {
         let timer = start_commit_timer("storage:apply_updates");
 
         let account_db = AccountDB::new(address_hash, self.db.clone());
-        let mut rocks_account = rocks_account.unwrap_or_default();
+        let mut rocks_account: RocksAccount = self
```

Comment:
I think this can be extracted into separate function, as we use it in multiple places (both reading the account from db and decoding it).